sopel_modules/weather/weather.py

```diff
@@ -150,7 +152,11 @@ def get_wind(speed, bearing):
 def convert_timestamp(timestamp, tz):
     time = datetime.utcfromtimestamp(timestamp)
     # We only return the time, without a date or timezone.
-    return format_time(time = time, zone = tz)[13:18]
+    # Taken from sopel/tools/time.format_time
+    utc = pytz.timezone('UTC')
+    time = utc.localize(time)
```

Comment:
Wouldn't it be easier to start with `time = datetime.fromtimestamp(timestamp, pytz.timezone('UTC'))`, rather than starting with a naïve `datetime` object that you then immediately convert to aware before using it? It's certainly possible to skip the `utc.localize()` step, and perhaps less confusing.

_(Stupid GitHub refusing to allow suggestions because the range of lines happens to contain a deletion.)_

Reply:
True, done.